app/services/appointment_notification_service.rb

```diff
@@ -7,7 +7,12 @@ def initialize(appointments:, days_overdue: 3, schedule_at:)
     @appointments = appointments
     @days_overdue = days_overdue
     @schedule_at = schedule_at
-    @communication_type = Communication.communication_types[:missed_visit_sms_reminder]
+
+    if FeatureToggle.enabled?("WHATSAPP_APPOINTMENT_REMINDERS")
+      @communication_type = Communication.communication_types[:missed_visit_whatsapp_reminder]
+    else
+      @communication_type = Communication.communication_types[:missed_visit_sms_reminder]
+    end
   end
 
   def send_after_missed_visit
```

Comment:
We could extract out `overdue_with_patient_consent` in its own scope on `Appointment`.

Reply:
I asked @prabhanshuguptagit to do it in https://github.com/simpledotorg/simple-server/pull/993 instead (might be a simpler change there)